### DISCUSSION TITLE
Is the cache invalid in production mode?

### DISCUSSION THREAD

**Discussion by eynait1010** [webpack/webpack#14674] [Q&A]
## Feature request
With wp4 4, it takes a minute to start my project, and now it only takes 6 seconds with webpack 5, which is awesome.  but the production mode has not become faster, so is the cache invalid in production mode?

**What is the expected behavior?**

**What is motivation or use case for adding/changing the behavior?**

**How should this be implemented in your opinion?**

**Are you willing to work on this yourself?**
yes

**Comment by alexander-akait** [comment]
Yes, we use different caches for development and production

**Reply by eynait1010** [reply]
What can i do to improve compile efficiency in production mode, I tried babel cache and eslint cache, it is still slow(60s+)

**Reply by alexander-akait** [reply]
Without configuration I can't say
